yarn.lock

```diff
@@ -2,9 +2,61 @@
 # yarn lockfile v1
 
 
+"@babel/code-frame@7.0.0-beta.32", "@babel/code-frame@^7.0.0-beta.31":
```

Comment:
I've polled our Slack, but I would like to _either_ commit `package-lock.json` or `yarn.lock` instead of maintaining both in future diffs. Open to discussion about this of course!